### ISSUE TITLE
line breaks in text with layer_text (bug and feature request)

### ISSUE THREAD

**Issue by coolbutuseless** [rstudio/ggvis#402]
While trying to add some text with a line break using layer_text, I found a bit of a bug - not sure if it's vega's problem or ggvis's problem.

The following code works fine.
```r
ggvis(data.frame(x=1:10, y=1:10), x = ~x, y = ~y) %>% layer_text(text := "hello")
```

Text with a "\n" in it causes an alert box to appear in Rstudio saying: `[Vega Err] SyntaxError: Unexpected EOF`
```r
ggvis(data.frame(x=1:10, y=1:10), x = ~x, y = ~y) %>% layer_text(text := "hello \n hello")
```

And trying to put a `<br/>` in the text does nothing because all the html gets escaped before display. i.e.
```r
ggvis(data.frame(x=1:10, y=1:10), x = ~x, y = ~y) %>% layer_text(text := "hello <br/> hello")
```
So my feature request is: enable multi-line printing with layer_text().

**Comment by ellisp** [comment]
Related to #359?

**Comment by coolbutuseless** [comment]
Yes.  Issue #359 covers my feature request to allow multi-line printing by allowing html (i.e. <br/>) tags to pass through.

But the `[Vega Err] SyntaxError` when string includes "\n" still seems to be a bug.

**Comment by vitorbl** [comment]
You could set two layers of text insted of one, try this:

ggvis(data.frame(x=1:10, y=1:10), x = ~x, y = ~y) %>% layer_text(text := "hello", dy :=-10) %>% layer_text(text := "world")
